src/custom/components/swap/EthWethWrap/WrapCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { CurrencyAmount, Currency } from '@uniswap/sdk'
+import styled from 'styled-components'
+import CurrencyLogo from 'components/CurrencyLogo'
+import { SHORT_PRECISION } from 'constants/index'
+
+const BalanceLabel = styled.p<{ background?: string }>`
+  display: flex;
+  justify-content: center;
+  margin: 0.5rem 0;
+  width: 100%;
+
+  border-radius: ${({ theme }) => theme.buttonPrimary.borderRadius};
+
+  background: ${({ background = 'initial' }) => background};
+
+  > span {
+    &:first-child {
+      margin-right: 0.2rem;
+    }
+  }
+`
+
+const WrapCardWrapper = styled.div`
+  ${({ theme }) => theme.flexColumnNoWrap}
+  align-items: center;
+  justify-content: center;
+  flex: 1;
+  padding: 0.5rem;
+`
+
+export const WrapCardContainer = styled.div`
+  position: relative;
+  ${({ theme }) => theme.flexRowNoWrap}
+  border: 0.15rem solid ${({ theme }) => theme.bg1};
```

Comment:
Think we only use pixels throughout the app?

Reply:
do we? or you mean for borders

Reply:
My impression is, that we are using px's everywhere, based on UNI inheritance (proper REM setup is superior though). But in the context of our (forked) front-end, I could see weird side effects when deviating from that.

Reply:
true